tests/introspection/ApiCtorInitTest.cs

```diff
@@ -145,6 +147,11 @@ protected virtual bool Skip (PropertyInfo pi)
 		/// <param name="type">Type of the object, to be used if special logic is required.</param>
 		protected virtual void Dispose (NSObject obj, Type type)
 		{
+			//***** ApiCtorInitTest.DefaultCtorAllowed
+			//2017-01-23 15:52:09.762 introspection[4084:16658258] *** -[NSKeyedArchiver dealloc]: warning: NSKeyedArchiver deallocated without having had -finishEncoding called on it.
```

Comment:
LGTM, if this cause issues elsewhere (now or in the future) we can add a `switch (type.Name)` and only call the selector for `NSKeyedArchiver`